ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -35,19 +35,22 @@ export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/parallel/normal"
 parallel_cucumber -n "${PARALLEL}" --first-is-1 --type cucumber --serialize-stdout --combine-stderr --prefix-output-with-test-env-number --exec \
     'export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS=$(echo ${USERS} | cut -d "," -f ${TEST_ENV_NUMBER},$((${TEST_ENV_NUMBER}+${PARALLEL})),$((${TEST_ENV_NUMBER}+${PARALLEL}*2)),$((${TEST_ENV_NUMBER}+${PARALLEL}*3)));
      export WORKSPACE=/tmp/dir${TEST_ENV_NUMBER};
-     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and not @admin and not @serial and ${E2E_SKIP_TAGS}\" -p junit"' || true
+     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and not @admin and not @serial and not @console and ${E2E_SKIP_TAGS}\" -p junit"' || true
 
 # run admin tests
 export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/parallel/admin"
 parallel_cucumber -n "${PARALLEL}" --first-is-1 --type cucumber --serialize-stdout --combine-stderr --prefix-output-with-test-env-number --exec \
     'export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS=$(echo ${USERS} | cut -d "," -f ${TEST_ENV_NUMBER},$((${TEST_ENV_NUMBER}+${PARALLEL})),$((${TEST_ENV_NUMBER}+${PARALLEL}*2)),$((${TEST_ENV_NUMBER}+${PARALLEL}*3)));
      export WORKSPACE=/tmp/dir${TEST_ENV_NUMBER};
-     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and @admin and not @serial and ${E2E_SKIP_TAGS}\" -p junit"' || true
+     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and @admin and not @serial and not @console and ${E2E_SKIP_TAGS}\" -p junit"' || true
 
-# run the rest tests in serial
-export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/serial"
 export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS="${USERS}"
+# run serial tests
+export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/serial"
 cucumber --tags "${E2E_RUN_TAGS} and @serial and ${E2E_SKIP_TAGS}" -p junit || true
+# run console tests
+export BUSHSLICER_REPORT_DIR="${ARTIFACT_DIR}/console"
```

Comment:
Is it ok to also put console junit results in the `${ARTIFACT_DIR}/serial` directory?

Reply:
Fine to me. Let's wait for the tests to finish then I'll make it this way.